packages/webapp/src/components/AddNewCrop/index.jsx

```diff
@@ -81,6 +85,40 @@ export default function PureAddNewCrop({
   const cropImageUrlRegister = register(CROP_PHOTO_URL, { required: true });
   const crop_photo_url = watch(CROP_PHOTO_URL);
 
+  const farmCropsSelector = useSelector(cropsOnMyFarmSelector);
```

Comment:
I think this should be `farmCrops` not selector.

Reply:
Yes it should, thank you 🙂